packages/nimble-components/karma.conf.js

```diff
@@ -49,7 +49,8 @@ module.exports = config => {
             'karma-source-map-support',
             'karma-sourcemap-loader',
             'karma-chrome-launcher',
-            'karma-firefox-launcher'
+            'karma-firefox-launcher',
```

Comment:
Can we make sure firefox is getting redirected to playwright's copy by updating the env var like we did chrome at the top: https://github.com/karma-runner/karma-firefox-launcher#environment-variables

Reply:
Updated